src/components/DpTooltip/DpTooltip.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpTooltip from './DpTooltip.vue'
+
+const meta: Meta<typeof DpTooltip> = {
+    component: DpTooltip,
+    title: 'Components/Tooltip',
+    render: (args) => ({
+        components: {
+            DpTooltip,
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-tooltip v-bind='args'>
+          my text
```

Comment:
Let's choose a text that is instantly recognisable as tooltip content, either with a basic, yet realistic example like `Click to cancel editing the item`, or with a more descriptive dummy text like `Tooltip content goes here`. I tend to favour the second one as it imposes less congitive load.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/505/commits/c16ba2ff8249ebc93299ef697e668d5bc5331534